app/views/classrooms/_classroom_students_table.html.erb

```diff
@@ -48,6 +51,15 @@
                 $0.00
               <% end %>
             </td>
+            <% if @can_manage_students %>
+              <td class="px-6 py-4 whitespace-nowrap text-sm text-gray-900 font-medium">
+                <% if student.portfolio %>
+                  $<%= number_with_precision(student.portfolio.cash_balance, precision: 2) %>
```

Comment:
I prefer the `number_to_currency`, because it handles the precision, the dollar sign, and it will help us if it needs to work with different currencies later. 

Usage for instance: 

https://github.com/rubyforgood/stocks-in-the-future/blob/main/app/views/stocks/index.html.erb#L16